velox/common/memory/SharedArbitrator.h

```diff
@@ -165,7 +165,6 @@ class SharedArbitrator : public memory::MemoryArbitrator {
   // Contains the execution state of an arbitration operation.
   struct ArbitrationOperation {
     MemoryPool* const requestPool;
-    MemoryPool* const requestRoot;
```

Comment:
Should we keep the name requestRoot instead of requestPool? Will be more intuitive and readable

Reply:
The request root is named to be different than the leaf pool. Now we always initiate from a root memory pool so we don't need to name it as root.